feature/security/gnsi/certz/client_certificates/README.md

```diff
@@ -0,0 +1,109 @@
+# X.X: gNSI Client Certificate Tests
+
+## Summary
+
+Clients must be able to validate a remote server's TLS certificate
+and present a valid client certificate to that server in order to provide
+identification information. The client certificate should have a
+SPIFFE Idenitifier embedded in it to be used as the identifier of
+the client to the server.
+
+## Baseline Setup
+
+### Input Args
+
+* the set of certificate testdata generated with the mk_cas.sh
+   script in featureprofiles/feature/security/certificate-authorties/
+
+### DUT service setup
+
+Configure the DUT to enable the following services (that are using gRPC) are
```

Comment:
I think we should also call out that: 
* Load server cert, CA bundle for gNSI service (i.e. how can we test gNSI.Certz if we can not even establish the first gNSI.Cert connection? :) ).

Reply:
I think that's this text:

```
Configure the DUT to enable the following services (that are using gRPC) are
up and using mTLS for authentication:

* gNMI
* gNOI
* gNSI
* gRIBI
* P4RT

Be prepared to load the relevant trust_bundle.pem file for each test
Certificate Authority(CA) under test on the DUT. Each CA has a RSA and ECDSA
form, both must be tested.

```

I believe that text is in 'all' of the test README files.